Sources/Subscription/Managers/StorePurchaseManager.swift

```diff
@@ -166,8 +166,6 @@ public final class DefaultStorePurchaseManager: ObservableObject, StorePurchaseM
         if self.purchasedProductIDs != purchasedSubscriptions {
             self.purchasedProductIDs = purchasedSubscriptions
         }
-
-        subscriptionGroupStatus = try? await availableProducts.first?.subscription?.status.first?.state
```

Comment:
Curious - Why is this not needed now? (mainly curious because we are changing production code)

Reply:
These properties were not used anywhere and were leftovers from the prototype where we checked this state. It is no longer relevant in the current implementation.